hub/core/chunk_engine.py

```diff
@@ -575,6 +577,8 @@ def _samples_to_chunks(
         nsamples = len(samples)
         if register and update_commit_diff:
             commit_diff = self.commit_diff
+        if progressbar:
```

Comment:
add a test with `progressbar` parameter True

Reply:
@FayazRahman I was thinking I should, but didn't know what the test would verify. What should I assert here? If the progress bar is displayed when the variable `progressbar` is True? Not sure how I should proceed here.

Reply:
just run with progressbar=True and assert that the samples have been added